node_topic: "/node"

lidar:
  min_range: 1.0
  max_range: 56.0

tgs:
  resolution: 8.0
  num_iter: 3
  num_lpr: 20
  num_min_pts: 10
  th_seeds: 0.3
  th_dist: 0.2
  th_normal: 0.940
  th_weight: 0.999
  th_lcc_normal: 0.03 
  th_lcc_planar: 0.1
  th_merging: 0.1
  sensor_height: 1.723
  th_seed_disparity: 0.2
  adaptive_seeds_margin: -1.2
  refine_mode: true
  visualization: true
  rgnr: true
  tnnr: true
  mnpf: true
